packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockEvolution.tsx

```diff
@@ -11,57 +11,33 @@ import {
   XAxis,
   YAxis,
 } from '@common/components';
-import { DateUtils } from '@common/utils';
 import {
   Box,
   useFormatDate,
   useTheme,
   useTranslation,
 } from '@openmsupply-client/common';
 import { DraftRequestLine } from '../hooks';
-import { useRequestFields } from '../../../api';
 
 export interface StockEvolutionProps {
   draftLine: DraftRequestLine | null;
 }
 
-type valueWithDate = {
-  date: Date;
-};
-const dateSorter = (a: valueWithDate, b: valueWithDate) => {
-  if (DateUtils.isEqual(a.date, b.date)) return 0;
-  if (DateUtils.isAfter(a.date, b.date)) return 1;
-  return -1;
-};
-
 export const StockEvolution: React.FC<StockEvolutionProps> = ({
   draftLine,
 }) => {
   const t = useTranslation('replenishment');
   const theme = useTheme();
   const d = useFormatDate();
-  const { maxMonthsOfStock, minMonthsOfStock } = useRequestFields([
-    'maxMonthsOfStock',
-    'minMonthsOfStock',
-  ]);
+  const data = draftLine?.chartData?.stockEvolution.nodes ?? [];
 
-  const data =
-    draftLine?.chartData?.stockEvolution.nodes
-      .map(node => ({
-        date: new Date(node.date),
-        value: node.historicStockOnHand ?? node.projectedStockOnHand ?? 0,
-        isHistoric: node.historicStockOnHand !== null,
-        max: draftLine.itemStats.averageMonthlyConsumption * maxMonthsOfStock,
-        min: draftLine.itemStats.averageMonthlyConsumption * minMonthsOfStock,
-      }))
-      .sort(dateSorter) ?? [];
-  const dateFormatter = (date: Date) =>
-    d(date, {
+  const dateFormatter = (date: string) =>
```

Comment:
for this one, the static data has `isHistoric` as true for all the later dates.. but I've kept the logic correct in the hope that this will be fixed when the query is changed to valid data